packages/web3-common/src/constants.ts

```diff
@@ -29,6 +29,7 @@ export const ERR_TX_NO_CONTRACT_ADDRESS = 403;
 export const ERR_TX_CONTRACT_NOT_STORED = 404;
 export const ERR_TX_REVERT_WITHOUT_REASON = 405;
 export const ERR_TX_OUT_OF_GAS = 406;
+export const ERR_TX_UNDEFINED = 407;
```

Comment:
```suggestion
export const ERR_RAW_TX_UNDEFINED = 407;
```

Reply:
Because this is only used for `UndefinedRAWTransactionError` I would change the name to be more explicit, since we might have a `ERR_TX_UNDEFINED` used for `web3-eth-tx` package